darwin/dataset/remote_dataset.py

```diff
@@ -235,6 +235,9 @@ def pull(
         ValueError
             If darwin in unable to get ``Team`` configuration.
         """
+
+        console = Console()
```

Comment:
See L120, there's a console built into this, which allows the user of these classes to define various console params, and although not used much _theoretically_ this is for allowing capturing of output to different forms etc.

```suggestion
        console = self.console or Console()
```